src/Dreamberd/Bytecode/Encode.hs

```diff
@@ -33,8 +33,8 @@ transpileString :: String -> [Char]
 transpileString str = transpileInt (length str) ++ str
 
 transpileCall :: Call -> [Char]
-transpileCall (Builtin op) = toEnum 0x21 : [toEnum (0x31 + fromEnum op)]
-transpileCall (FunctionName str) = toEnum 0x22 : transpileString str
+transpileCall (Operator op) = toEnum 0x21 : [toEnum (0x31 + fromEnum op)]
```

Comment:
Why operator ?

Reply:
Builtin represents a function (atm print, error or input)
While Operator is the list of the operators we support (+, -, *, etc)